src/Pos/Generator/Block/Param.hs

```diff
@@ -39,16 +42,46 @@ instance Buildable AllSecrets where
             (length _asSecretKeys)
             (HM.keys _asSecretKeys)
 
+-- | Parameters for transactions payload generation.
+data TxGenParams = TxGenParams
```

Comment:
Why would it be it `txp`? There's not much code and it's easier to read it when it's all in one place. I do not see any reason to move it outside for now.